apps/lockfile-explorer/config/heft.json

```diff
@@ -33,17 +33,6 @@
           }
         }
       }
-    },
-
-    "serve": {
-      "phaseDependencies": ["build"],
-      "tasksByName": {
-        "express": {
-          "taskEvent": {
-            "eventKind": "nodeService"
-          }
-        }
-      }
```

Comment:
Why remove this code path? Most of the time this project is built no one will be running the service, so including the plugin causes unnecessary extra overhead on the build. I expect in the near future to be moving webpack into its own phase in the projects that have a webpack compilation, as well.

Reply:
I don't have a strong opinion about that, except that the `--serve` semantics should be identical between Webpack dev server and `NodeServicePlugin`, since from an end user's perspective they are doing exactly the same thing.

@D4N14L seemed to prefer the `heft build-watch --serve` design versus `heft serve-watch`.  I don't have a strong opinion about it, except that we need to be consistent.  

Also after https://github.com/microsoft/rushstack/pull/4168 brings back `heft start`, I think people will prefer `heft start` and so these implementation details perhaps won't be very visible to regular users.

Reply:
I tend to agree with Pete on this one. I think consistency is key on this. The webpack plugin is different in that bundle/serve are both handled by the same tool, whereas this one is serve-specific. If we want a consistent flow, then it would make sense to include the serve functionality in the same step that produces the files being served.